ESMF/GOCART2G_GridComp/CA2G_GridComp/CA2G_GridCompMod.F90

```diff
@@ -169,7 +169,7 @@ subroutine SetServices (GC, RC)
 !   ------------------------
     call MAPL_GridCompSetEntryPoint (GC, ESMF_Method_Initialize,  Initialize, __RC__)
     call MAPL_GridCompSetEntryPoint (GC, ESMF_Method_Run, Run, __RC__)
-    if (data_driven /= .true.) then
+    if (data_driven .neqv. .true.) then
```

Comment:
Am wondering if the following isn't clearer anyway given on of the operands is constant:
```
if (.not. data_driven) then
```
To my mind this is somewhat easier to grok.

Reply:
And now that I see the other changes are duplicates of this,  it begs the introduction of a new logical variable:
```
logical :: not_data_driven
...
not_data_driven = .not. data_driven
...
if (not_data_driven) then
...
```

The question becomes, is there a better name than `not_data_driven`.  Ideally something more active/positive so that the "not" disappears entirely.   I'll ask anton.

Reply:
I checked with Elliot, and the only antonym he could think of was `computational`.

Thinking ...

Reply:
I suppose `model` gets the point across, but somehow it still does not sound right.

Reply:
After a bit of googling, I think the term `numerical` is reasonably common:
```
logical :: numerical ! as opposed to data-driven
...
numerical = .not. data_driven
...
if (numerical) then
...
```